registry/client/errors/errors_test.go

```diff
@@ -0,0 +1,104 @@
+package errors
```

Comment:
As the old errors are now an alias for the ones that are removed, I'd suggest removing the original `errors_test.go` (i.e., moving from `registry/client` to `registry/client/errors`)